dp-cluster-setup-utility.py

```diff
@@ -1213,67 +1616,216 @@ def satisfied(self):
 
   def current_user(self):
     return pwd.getpwuid(os.getuid()).pw_name
+"""
+  Class for controlling flow of execution 
+"""
 
-if __name__ == '__main__':
-  user = Memorized(User())
-  print '\nThis script will check to ensure that all necessary pre-requisites have been met and then register this cluster with DataPlane.\n'
-  print 'Please ensure that your cluster has kerberos enabled, Ambari has been configured to use kerberos for authentication, and Knox is installed. Once those steps have been done, run this script from the Knox host and follow the steps and prompts to complete the cluster registration process.\n'
-
-  if not ScriptPrerequisites().satisfied():
-    sys.exit(1)
-
-  print 'Tell me about your DataPlane Instance'
-  dp = DataPlane(user.url_input('DataPlane URL', 'dp.url'), user.credential_input('DP Admin', 'dp.admin'))
+class FlowManager(object):
+  def __init__(self, cluster_type):
+    self.cluster_type = cluster_type
+    self.flow_manager = None
```

Comment:
Perhaps you should call this operations_manager. This class itself is FlowManager. So this variable name is confusing.